docs/source/conf.py

```diff
@@ -39,19 +40,65 @@
 #
 # needs_sphinx = '1.0'
 
-# Add any Sphinx extension module names here, as strings. They can be
+# Add any Sphinx extension module names here as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
```

Comment:
The list below I tried to match up to the list for MAPDL. I did not remove an extension that was unique to DPF-Core.

Reply:
Agree with most, but made a few suggestions.